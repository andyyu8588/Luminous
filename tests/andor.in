print(true and false);
print(true or false);
print(true and true);
print(false or false);
print(true and true or false or true or false and false);
print(false or false or false or true and false);