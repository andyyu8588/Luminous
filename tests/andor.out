false
true
true
false
true
false
